app/scripts/models/reliers/relier.js

```diff
@@ -102,6 +102,12 @@ define(function (require, exports, module) {
           } else {
             self.importSearchParam('email');
           }
+
+          self.importSearchParam('migration');
+          if (self.has('migration')) {
```

Comment:
I had a chat with @rfk today about normalizing OAuth params. His feeling is `don't do it.` Let's take out the normalization here.